### DISCUSSION TITLE
Another ML Project

### DISCUSSION THREAD

**Discussion by ozmosys** [openstates/issues#669] [Q&A] [answered]
Hello,

We want to test our ML project on legislative data. We have trained a few models on privacy and discrimmination. 

Can you help us with the API logic to produce the most recent introduced and passed legislation for the current sessions across all States? 

Thanks in advance.

Eric

**Comment by NewAgeAirbender** [comment, marked as answer]
Hello,

Depending on what format you want the data in, maybe the [bulk data downloads](https://openstates.org/data/) would be a great place to start since almost half of the states' 2022 sessions are adjourned for the year. Otherwise, I'd recommend experimenting with queries/responses on the [interactive documentation](https://v3.openstates.org/docs) for V3 to see how to best do that. 

Best of luck!
Rylie
